<ui version="4.0" >
 <class>MotorWidget</class>
 <widget class="QWidget" name="MotorWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>228</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="horizontalSpacing" >
    <number>4</number>
   </property>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string/>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="directionLabel" >
        <property name="text" >
         <string>L/R Motor:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="mneComboBox" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="contextMenuPolicy" >
         <enum>Qt::ActionsContextMenu</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="curPosLabel" >
        <property name="text" >
         <string>Current Position:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QDoubleSpinBox" name="posSpinBox" >
        <property name="minimumSize" >
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="contextMenuPolicy" >
         <enum>Qt::ActionsContextMenu</enum>
        </property>
        <property name="wrapping" >
         <bool>false</bool>
        </property>
        <property name="buttonSymbols" >
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals" >
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="stepSizeLabel" >
        <property name="text" >
         <string>Step Size:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QDoubleSpinBox" name="stepSpinBox" >
        <property name="minimumSize" >
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="cursor" >
         <cursorShape>IBeamCursor</cursorShape>
        </property>
        <property name="contextMenuPolicy" >
         <enum>Qt::ActionsContextMenu</enum>
        </property>
        <property name="buttonSymbols" >
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="suffix" >
         <string/>
        </property>
        <property name="decimals" >
         <number>2</number>
        </property>
        <property name="minimum" >
         <double>-100.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>100.000000000000000</double>
        </property>
        <property name="value" >
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2" >
       <widget class="Line" name="line" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="curPosLabel_2" >
        <property name="text" >
         <string>Next Position:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QDoubleSpinBox" name="nextPosSpinBox" >
        <property name="minimumSize" >
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="contextMenuPolicy" >
         <enum>Qt::ActionsContextMenu</enum>
        </property>
        <property name="buttonSymbols" >
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="prefix" >
         <string/>
        </property>
        <property name="decimals" >
         <number>2</number>
        </property>
        <property name="minimum" >
         <double>-2000000000.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>2000000000.000000000000000</double>
        </property>
        <property name="defaultValue" stdset="0" >
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2" >
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <property name="sizeConstraint" >
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="lowLimitLabel" >
          <property name="text" >
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="nextPosSlider" >
          <property name="contextMenuPolicy" >
           <enum>Qt::ActionsContextMenu</enum>
          </property>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="highLimitLabel" >
          <property name="text" >
           <string>100</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0" colspan="2" >
       <spacer name="verticalSpacer" >
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionSaveStartLocation" >
   <property name="text" >
    <string>Save Start Location</string>
   </property>
   <property name="autoRepeat" >
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSaveStopLocation" >
   <property name="text" >
    <string>Save Stop Location</string>
   </property>
   <property name="autoRepeat" >
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
